pages/linux/imgp.md

```diff
@@ -0,0 +1,15 @@
+# imgp
+
+> Command line image resizer and rotator for JPEG and PNG images.
+
+- Convert single images and/or whole directories containing valid image formats:
+
+`imgp -x {{1366x1000}} {{path/to/dir}} {{path/to/file}}`
+
+- Scale an image by 75% and overwrite the source image:
+
+`imgp -x {{75}} -w ~/image.jpg`
```

Comment:
s/~/image.jpg/{{path/to/image}}

Reply:
Is there a reason you have changed this to path/to/directory without changing the description ? The description mentions "source image", hence I mentioned "path/to/image". I don't care about "image" or "directory". But the description and the example has to be consistent.

Reply:
my mistake